api/actions/actions-router.js

```diff
@@ -1 +1,237 @@
-// Write your "actions" router here!
+const router = require('express').Router()
+const Actions = require('./actions-model')
+const { validateActionsId, validateAction } = require('./actions-middlware')
+
+
+router.get('/', (req,res,next) =>{
+    Actions.get()
+            .then(actions=>res.json(actions))
+            .catch(err => next(err))
+})  
+
+// not sure if I understand the catch-next part, I just use it
+// kind of how we don't really know how airplanes work yet fly
+// please leave feedback if you read this
```

Comment:
next is just sending your function to the next function in your server. In your case it will just be the server catch in your server,js file. You would ideally want a error handler to catch all errors from your router.